heron/spi/src/java/com/twitter/heron/spi/common/ShellUtils.java

```diff
@@ -92,10 +89,12 @@ public static int runSyncProcess(
       }
       pStdOut.append(inputstreamToString(process.getInputStream()));
       pStdErr.append(inputstreamToString(process.getErrorStream()));
-      if (verbose) {
-        LOG.info(pStdOut.toString());
-        LOG.info(pStdErr.toString());
-      }
+
+      LOG.log(Level.FINE, "\tSTDOUT: {0}", pStdOut.toString());
```

Comment:
We determine whether the logging level is Level.FINE or Level.INFO, basing whether a verbose flag is passed. 

By design, we don't want to show the stdout of a spawned process unless a verbose flag is passed, since:
1. We should provide more meaningful INFO in itself, rather than rely on the stdout of a spawned process.
2. Also it is more controllable, since we have not ideas what the stdout can be. (potentially very large as you mentioned)

But we want to show stederr no matter the verbose flag being passed or not, since it is more considered as a kind of error message. (Level.SERVER)